Notebook.ipynb

```diff
@@ -194,7 +216,8 @@
    "source": [
     "Let's see a small demo of our project. In order to run the following code, and in general to replicate our work, make sure that:\n",
     "1. this notebook stays inside the folder in which it was delivered. This is necessary to resolve the imports\n",
-    "2. the following packages that were used in the hands-on sessions are installed: `pyswip`, `minihack`, `numpy`, `nle`, `matplotlib`, `notebook` "
+    "2. the following packages that were used in the hands-on sessions are installed: `pyswip`, `minihack`, `numpy`, `nle`, `matplotlib`, `notebook` \n",
+    "3. the `pandas` package is installed, in order to plot the data in the assessment paragraph"
```

Comment:
May be needed to install another package for dealing with excels ..
i don't remember how it's called tho ... hehe